lib/satellite/controller_extensions.rb

```diff
@@ -84,10 +94,33 @@ def skip_satellite_authentication?
       !!@skip_satellite_authentication
     end
 
+    def skip_satellite_authentication
+      @skip_satellite_authentication = true
+    end
+
+    def should_skip_satellite_authentication?
+      (params[:skip].to_i == 1).tap { |skip| delete_user_identity_cookie if skip }
```

Comment:
Since the value "1" matters, maybe extract a constant at the top of the file that communicates that?